ccd-definition/AuthorisationCaseField/CareSupervision/judiciary.json

```diff
@@ -2725,7 +2725,14 @@
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "CaseFieldID": "sdoDirectionCustom",
+    "CaseFieldID": "customDirections",
+    "UserRole": "caseworker-publiclaw-judiciary",
+    "CRUD": "CRU"
```

Comment:
I changed it in my PR to be CRUD, needs to be CRUD so user can delete previously entered draft directions

Reply:
cool, changed